tests/src/Interop/StructMarshalling/PInvoke/MarshalStructAsParamDLL.h

```diff
@@ -41,13 +39,7 @@ bool IsCorrectInnerSequential(InnerSequential* p)
 		return false;
 	if(p->f2 != 1.0)
 		return false;
-
-	char const * lpstr = "some string";
-	size_t size = sizeof(char) * (strlen(lpstr) + 1);
-	LPSTR temp = (LPSTR)TP_CoTaskMemAlloc( size );
-	memset(temp, 0, size);
```

Comment:
This basically creates an empty string that takes `size` bytes of storage for no reason, the string is only used as an argument to `strcmp`.